student-store-ui/src/components/AddToCartButton/AddToCartButton.jsx

```diff
@@ -0,0 +1,40 @@
+import { useContext } from "react";
+import "../AddToCartButton/AddToCartButton.css";
+import { CheckoutCartContext, CheckoutCartDispatchContext } from "../CheckoutCartContext/CheckoutCartContext";
+export default function AddToCartButtton({ product, checkoutCart, dispatch }){
+    // const checkoutCart = useContext(CheckoutCartContext); // cart context
+    // const dispatch = useContext(CheckoutCartDispatchContext);
```

Comment:
I like your approach of using useContext, feel free to dive into this further as it'll help in reducing the amount of prop drilling you need to make for future projects